README.md

```diff
@@ -51,7 +51,7 @@ Or expanding an abbreviation in text:
 
 ```rust
 let parser = parser.map(|event| match event {
-	Event::Str(text) => Event::Str(text.replace("abbr", "abbreviation")),
+	Event::Text(text) => Event::Str(text.replace("abbr", "abbreviation")),
```

Comment:
Shouldn't the `Event::Str` on the RHS also be changed?

Reply:
Whoops, fixed!